source/drivers/java.txt

```diff
@@ -0,0 +1,234 @@
+.. include:: /includes/unicode-checkmark.rst
+
+===================
+Java MongoDB Driver
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+Introduction
+------------
+
+The official MongoDB Java Driver providing both synchronous and asynchronous interaction with MongoDB.
+
+- `Tutorials <http://mongodb.github.io/mongo-java-driver/3.10/driver/tutorials/>`__
+
+- `Usage Guide <http://mongodb.github.io/mongo-java-driver/3.10/driver/>`__
+
+- `API Documentation <http://mongodb.github.io/mongo-java-driver/3.10/javadoc/overview-summary.html>`_
+
+- `Changelog <http://mongodb.github.io/mongo-java-driver/3.10/whats-new/>`__
+
+- `Source Code <https://github.com/mongodb/mongo-java-driver/>`__
+
+
```

Comment:
If we link to the Connect guide (docs.mongodb.com/guides/drivers) here we get the dual benefit of taking the burden of connection tutorials off the driver page and also providing an incentive to make sure the guide stays current (the more we duplicate content, the more likely it is to get out of date somewhere on the site).